sway/input/keyboard.c

```diff
@@ -349,51 +394,25 @@ static void handle_key_event(struct sway_keyboard *keyboard,
 	bool shortcuts_inhibited = sway_inhibitor && sway_inhibitor->inhibitor->active;
 
 	// Identify new keycode, raw keysym(s), and translated keysym(s)
```

Comment:
I *think* this comment should be carried over to `update_keyboard_state`?

Reply:
Copied the comment